windows/hid.c

```diff
@@ -330,7 +330,7 @@ static struct hid_device_info *hid_get_device_info(const char *path, HANDLE hand
 	if (path) {
 		size_t len = strlen(path);
 		dev->path = (char*)calloc(len + 1, sizeof(char));
-		strncpy(dev->path, path, len + 1);
+		strcpy(dev->path, path);
 		dev->path[len] = '\0';
```

Comment:
an alternative:
```suggestion
		memcpy(dev->path, path, len + 1);
```
but I don't believe it makes any practical difference

Reply:
Actually this is better. It will also remove the MSVC warning 4996 against strcpy or strncpy.

```
warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, 
use _CRT_SECURE_NO_WARNINGS. See online help for details.
```

Reply:
Just pushed new commits which change to memcpy and remove the MSVC pragma to disable warning 4996.